Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1680,6 +1682,8 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
         model.primitiveIDOffset += cabo.getElementCount() / stride;
       }
     }
+    // reset the line width
+    gl.lineWidth(1);
```

Comment:
hi @martinken.
Unless I misunderstanding something, wouldn't this undo L:1664 and set the line width to `1` anyway?

Reply:
The lines are drawn between the two calls. We reset it to 1 so that it doesn't impact other mappers etc.

Reply:
got it. Makes sense, thanks for the explanation.